services/events/events_definitions/schemas.go

```diff
@@ -0,0 +1,137 @@
+package events_definitions
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+var validOperator = []string{"EQUAL_TO"}
+var source = []string{"COMMAND_STATE_CHANGED", "MONITOR_TRIGGERED", "PASS_AOS", "PASS_LOS", "STREAM_DECODED"}
+var state = []string{"ACTIVE", "INACTIVE"}
+
+var eventsDefinitions = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"description": {
+		Type:     schema.TypeString,
+		Optional: true,
+	},
+	"mappings": {
+		Type:     schema.TypeSet,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: mappingSchema,
+		},
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"rules": {
+		Type:     schema.TypeSet,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: ruleSchema,
+		},
+	},
+	"source": {
+		Type:         schema.TypeString,
+		Required:     true,
+		Description:  "Allowed values: COMMAND_STATE_CHANGED, MONITOR_TRIGGERED, PASS_AOS, PASS_LOS, STREAM_DECODED",
```

Comment:
you can use `helper.AllowedValuesToDescription(source)` instead of hard-coding the list again :)